### PR TITLE
Djangosetup

### PR DESCRIPTION
**Pull request by luciencd**
Added a Django skeleton web app that holds a restapi sub app. 
Contains basic models representing representatives, legislation, etc...

Added extra documentation in main readme, src readme, and restapi django webapp readme.

### CODE REVIEW

File: server/web/code/src/rest/rest/settings.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+WSGI_APPLICATION = 'rest.wsgi.application'
+
+
+# Database
+# https://docs.djangoproject.com/en/1.10/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
```
Comment (SaswatB) [inline server/web/code/src/rest/rest/settings.py]: Why are we using sqllite here?
I believe we agreed on using MySQL.
Also I don't believe the sqllite file should not be commited, but I could not comment on that.

File: server/web/code/src/rest/rest/settings.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+USE_L10N = True
+
+USE_TZ = True
+
+
+# Static files (CSS, JavaScript, Images)
+# https://docs.djangoproject.com/en/1.10/howto/static-files/
+
+STATIC_URL = '/static/'
```
Comment (SaswatB) [inline server/web/code/src/rest/rest/settings.py]: I don't believe Django has any static assets

File: server/web/code/src/rest/rest/urls.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+Including another URLconf
+    1. Import the include() function: from django.conf.urls import url, include
+    2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
+"""
+from django.conf.urls import url,include
+from django.contrib import admin
+
+urlpatterns = [
+    url(r'^admin/', admin.site.urls),
+    url(r'^restapi/',include('restapi.urls')),
```
Comment (SaswatB) [inline server/web/code/src/rest/rest/urls.py]: I guess this is up for debate but would api/ be preferable to restapi/?

File: server/web/code/src/rest/rest/settings.py (hunk 1 above)
Comment (luciencd) [inline server/web/code/src/rest/rest/settings.py]: That was the default, I will go back and change it to the actual mysql server that will be running.

File: server/web/code/src/rest/rest/settings.py (hunk 2 above)
Comment (luciencd) [inline server/web/code/src/rest/rest/settings.py]: Unsure about that too, that was just in the default code.

File: server/web/code/src/rest/rest/settings.py (hunk 1 above)
Comment (SaswatB) [inline server/web/code/src/rest/rest/settings.py]: Ok, the compseexample settings file in the docker branch should have a mysql config example

**Pull request merged**